index.rst

```diff
@@ -1,83 +1,144 @@
 
-   Feel free to delete this instructional comment.
 
-:tocdepth: 1
+At first glance, it is apparent that dipoles are more prevalent in the g-band difference image of both simulations, which is due to three factors: 
 
-In order to test Differential Chromatic Refraction (DCR) correction algorithms we first need a metric that quantifies the severity of residual dipoles in difference images, in order to evaluate the performance of the correction. 
-I used the StarFast simulator (http://dmtn-012.lsst.io, https://github.com/lsst-dm/starfast_simulator) to generate multiple realizations of a small field at u, g, and r bands and across a range of airmasses from 1.0 to 2.0.
-I then used ingestSimImages.py and processEimage.py from obs_lsstSim to generate standard calibrated exposures, and ran standard LSST image differencing using imageDifference.py with a calibrated exposure near zenith used as the template (using GetCalexpAsTemplateTask).
+#. The g-band filter is twice as wide in wavelength as u-band, balancing the larger change in index of refraction at u-band (see Figure 1 of https://github.com/isullivan/LSST-DCR/blob/master/DCR%20literature%20overview.pdf)
+#. Most stars of type A or cooler have a uniformly rising spectrum in u-band, which results in a similarly dispersed psf (see the dipole images of individual stars at the bottom of https://github.com/lsst-dm/starfast_simulator/blob/master/StarFast_example.ipynb). In g-band, by contrast, a typical type A star will have a falling spectrum, a type K star will have a rising spectrum, and in between stars will have a spectrum that is on average flat, but full of features. As a result, there is a greater variation in PSF shape due to DCR in g-band than in any other band.
+#. Related to the above two points, since the SEDs of many stars peak in g-band, and because the bandwidth is wider, most stars are brightest in g-band. With brighter stars, we should expect to see more dipoles above the noise floor.
 
-.. figure:: /_static/Dipoles_ds9_fourpanel_f012.png
-   :name: fig-dipole_fourpanel
-   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipoles_ds9_fourpanel_f012.png
+Defining a Dipole Severity Metric
+=================================
+
+In order to leverage the full utility of LSST software, I wanted to use only the source lists generated by source detection and measurement in processEimage.py and the DIA sources generated with imageDifference.py. A metric of dipole severity could be defined a great many different ways from the parameters in the source tables, but after extensive trials and testing the following, though far from perfect, appears to be robust:
+
+* First, define a reference flux to use as a normalization for the full catalog of sources extracted in standard source detection and measurement.
+
+* Next, calculate the total significance of both positive and negative sources in the difference image, and divide by the uncertainty in their flux.
```

Comment:
I think it might be a better measure to include some kind of false positive measure. Perhaps that is along the lines of what you have already (just flux-weighted) but I am not sure. Do you include a measure of fluxes of sources which did not lead to dipoles (i.e. perfectly subtracted)?